_data/calendar.yml

```diff
@@ -560,3 +560,9 @@
   datum: "2023-05-03"
   uhrzeit: "20.00h"
   ort: "Raum Flex Office 2, 3.OG, Collective Incubator, Jülicher Str. 209d, 52070 Aachen"
+
+- title: Symposium
+  datum: "2023-05-13"
+  uhrzeit: "14.00h"
```

Comment:
To be discussed, aber lieber jetzt schonmal nen groben Blocker im Kalender drin haben...

Reply:
Passt